### PR TITLE
Add description + OID to metric help strings 

### PR DESCRIPTION
**Pull request by Conorbro**
Implements #130

### CODE REVIEW

**Comment by brian-brazil** [comment]
This looks good. You'll also need to regenerate snmp.yml.

File: snmp.yml [hunk 1]
```diff
@@ -2606,147 +3270,206 @@ ddwrt:
...
+    help: The number of 'ticks' (typically 1/100s) spent processing software interrupts
+      - 1.3.6.1.4.1.2021.11.61
   - name: ssRawSwapIn
     oid: 1.3.6.1.4.1.2021.11.62
     type: counter
+    help: Number of blocks swapped in - 1.3.6.1.4.1.2021.11.62
   - name: ssRawSwapOut
     oid: 1.3.6.1.4.1.2021.11.63
     type: counter
-  - name: ssCpuRawSteal
```
Comment (Conorbro) [inline snmp.yml]: I'm generating the yaml using the latest mibs as per the [generator readme]( http://www.net-snmp.org/docs/mibs/UCD-SNMP-MIB.txt) which has these metrics listed yet they are removed for some reason.

**Comment by Conorbro** [comment]
@brian-brazil

File: generator/net_snmp.go [hunk 2]
```diff
@@ -28,6 +28,7 @@ type Node struct {
 	Hint        string
 	Units       string
 	Access      string
+	Help        string
```
Comment (brian-brazil) [inline generator/net_snmp.go]: This shouldn't be here, it's already the Description.

File: config/config.go [hunk 3]
```diff
@@ -141,6 +141,7 @@ type Metric struct {
 	Name    string    `yaml:"name"`
 	Oid     string    `yaml:"oid"`
 	Type    string    `yaml:"type"`
+	Help    string    `yaml:"help"`
```
Comment (brian-brazil) [inline config/config.go]: The new field should be mentioned in the generator readme.

**Comment by Conorbro** [comment]
@brian-brazil

**Comment by brian-brazil** [comment]
Thanks!

**Comment by RichiH** [comment]
Nitpick: Please use https://help.github.com/articles/closing-issues-using-keywords/ as merging into master will close the referenced issues automagically.

**Comment by RichiH** [comment]
Personally, I use

`github.com closes #123`

as that allows non-github-users to see what the number is about.

**Comment by Conorbro** [comment]
No worries, will do so in future ! 👍

**Pull request merged**